### DISCUSSION TITLE
probleme avec qb64pe 314 et windows 10 32 et 64bits

### DISCUSSION THREAD

**Discussion by daniel71fr** [QB64-Phoenix-Edition/QB64pe#586] [Q&A]
Bonjour,
Quand j'essai d'ouvrir qb64pe 314 dans w10 en 32 ou 64 c'est impossible, seul la version 311   
fonctionne, je voudrai utiliser la 314 , comment résoudre ce problème ?
Merci  d'avance
Hello When I try to open qb64pe 314 in w10 in 32 or 64 it's impossible, only the 311 version works, I would like to use the 314, how to solve this problem? Thanks in advance

**Comment by a740g** [comment]
Can you try downloading the latest v4.0.0 from https://github.com/QB64-Phoenix-Edition/QB64pe/releases/latest to a new directory and try again?
